experiments/llm-eval/tasks/beta-promote/BigQueryReservationReservation-promote/cleanup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+REPO_DIR="$1"
+DEST_PATH="$2"
+
+if [ -d "${REPO_DIR}" ]; then
```

Comment:
I don't really understand what we are doing here, this is because you want to keep the temp directories?  This doesn't feel specific to the `[BigQueryReservationReservation-promote` task, which is why I don't think it should be here